e3sm_to_cmip/util.py

```diff
@@ -864,15 +864,20 @@ def find_mpas_files(component, path, map_path=None):
     component = component.lower()
     contents = os.listdir(path)
 
+    print(f"INFO: find_mpas_files: path = {path}")
+    logger.info(f"find_mpas_files: path = {path}")
+
     if component == "mpaso":
 
         pattern = r"mpaso.hist.am.timeSeriesStatsMonthly.\d{4}-\d{2}-\d{2}.nc"
         results = [
             os.path.join(path, x)
             for x in contents
-            if re.match(pattern=pattern, string=x)
+            if re.match(pattern=pattern, string=x) or re.match(pattern=pattern, string='.'.join(x.split('.')[3:]))
```

Comment:
@TonyB9000, could you add a comment here explaining this second term?  I believe we now prepend the name of the simulation.  It the issue that the name of the simulation contains dots?  If so, I wonder how robust this will be.

Reply:
I think the more robust solution would be to add a `.*` or something at the beginning of the pattern.  I'm not super good with regular expression but there's definitely a pattern like `.*` for any number of characters.

Reply:
I was looking into that, thanks.  This was quick and dirty, and assumes the "case-name" prefix always has 3 components, which is not terribly robust.  I'll conduct a test for eliding initial characters.

Reply:
Regarding the ncremap calls (in mpas.py), I'm not sure how best to produce a representative sample of command lines.

The additions of "-P <pcode>" are required by Zender, unless the mapfile name contains the string "sgs".  I don't know if this is a no-op for v1 usage, so I apply it everywhere.  I had to instrument the cmor handlers for mpaso and mpassi variables in order to issue the switches for "-P mpasocean" and "-P mpasseaice", etc.

Reply:
Maps with names `map*sgs*.nc` should no longer be used. Period. Use the `-P model_name` formulation instead, and replace any map-file names of the form `map*sgs*.nc` with the corresponding map-file that has no `sgs` in the name. The resulting commands will work on both v1 and v2 files.

Reply:
@czender Thanks! Especially good to know I don't need to switch on v1 vs v2 for this.

An issue with the e3sm_to_cmip repo is that the "scripts/cwl_workflows/*" contain <job>.yaml scripts, providing input parameters that are out of date and often reference people's personal directories.  I will attempt to clean that material as an adjunct to ensuring "-P model_name" is applied where appropriate (and eliminating use of "*sgs*" maps.)